devel/fuzz/Portfile

```diff
@@ -1,21 +1,28 @@
-PortSystem 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-name            	fuzz
-version         	0.6
-revision		1
-categories      	devel security
-license         	GPL-2
-maintainers     	nomaintainer
-platforms       	darwin
-description     	software security checking tool
-homepage        	http://fuzz.sourceforge.net/
-master_sites    	sourceforge:project/fuzz/fuzz/${version}
-checksums     		md5 8c8e7c49729e0a98c0414faac7778ec7
-configure.args		--mandir=${prefix}/share/man
-long_description 	The fuzz generator is designed to attack certain kinds \
-			of software and expose one particular kind of bug \
-			common in software. This is the situation where the \
-			programmer implicity makes some assumtions about \
-			the data stream that the program will be parsing.  \
-			If the data stream is substantially different then \
-			the program might not be able to deal with it.
+PortSystem          1.0
+
+name                fuzz
+version             0.6
+revision            2
+categories          devel security
+license             GPL-2
```

Comment:
your first commit changes this to "GPL-1" the second one back to "GPL-2"

Reply:
Sorry, will fix this.